tests/api/routers/test_role.py

```diff
@@ -22,13 +22,16 @@ def test_get_role_by_name(create_iam_role, iam_role_name) -> None:
     assert data["RoleName"] == iam_role_name
 
 
-def test_get_managed_policies_for_role(iam_role_name, attach_iam_role_policy, iam_policy_name):
+def test_get_policies_for_role(
+    iam_role_name, attach_iam_role_policy, attach_inline_policy, iam_policy_name
+):
     response = client.get(f"/roles/{iam_role_name}/policies/")
     data = response.json()
+    print(data)
 
-    assert len(data) == 1
+    assert len(data) == 2
     assert response.status_code == 200
-    assert data[0]["PolicyName"] == iam_policy_name
+    assert data[0]["PolicyName"] == "inline-policy"
```

Comment:
Could you do an assert on the second policy to check it is `iam_policy_name`?

Reply:
Agreed